### PR TITLE
Fix / remove profiles from the web app

### PR DESCRIPTION
**Pull request by borkopetrovicc**
## Description
[
](https://jwplayer.atlassian.net/jira/software/c/projects/PS/boards/575?selectedIssue=OWA-78)

https://www.figma.com/design/qLl8KxBOOr640h18ICx0H6/Web-app?node-id=3004-9048&t=RpZJoBJfnVp5w6Us-0

1.Disable "Export account" feature for JWP
2.Disable "Delete account" feature for JWP
3.Disable "Switch plans" for JWP

This PR solves # .

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: packages/ui-react/src/components/Payment/Payment.tsx [hunk 1]
```diff
@@ -221,19 +220,12 @@ const Payment = ({
...
+                    className={styles.changePlanCancelButton}
+                    label={t('user:payment.cancel_subscription')}
+                    onClick={onCancelSubscriptionClick}
+                    variant="danger"
                     disabled={!canRenewSubscription && activeSubscription.status === 'cancelled'}
-                    onClick={() => {
-                      if (offers.length > 1 && !canRenewSubscription) {
-                        setIsChangingOffer(true);
-                      } else {
-                        onUpgradeSubscriptionClick?.();
```
Comment (AntonLantukh) [inline packages/ui-react/src/components/Payment/Payment.tsx:231]: We can't remove this part, it is part of the Cleeng flow.

Comment (borkopetrovicc) [reply packages/ui-react/src/components/Payment/Payment.tsx:231]: can you check it again ?

File: packages/ui-react/src/components/Payment/Payment.tsx [hunk 2]
```diff
@@ -219,23 +221,34 @@ const Payment = ({
...
-                      }
-                    }}
-                    fullWidth={isMobile}
-                    color="primary"
-                    data-testid="change-subscription-button"
-                  />
-                )
+                <>
+                  {canSwitchSubscription && showChangeSubscriptionButton && (
```
Comment (AntonLantukh) [inline packages/ui-react/src/components/Payment/Payment.tsx:225]: I am hesitant to add additional complexity here introducing a new condition + new "cancel" button (third one?). I like the solution Christiaan purposed. Do you think we could focus on that one?

Comment (borkopetrovicc) [reply packages/ui-react/src/components/Payment/Payment.tsx:225]: Yes i will close this pr.

**Pull request closed without merging**